src/frontend/Shared/components/App.tsx

```diff
@@ -1,9 +1,13 @@
 import * as React from 'react';
-import {CBUTTON} from './Button';
+import Sidebar from './Sidebar';
+import FriendList from './FriendList'
 
 const App = () => {   //this is how you make a functional component
   return (
-      <div><CBUTTON /></div>
+      <div>
+        <Sidebar />
+        <FriendList />
```

Comment:
Do we need to make this component? Should they be integrated to other components individually?

Reply:
It should be in its component, but it should be a general component that all other tasks can use